pkg/nsx/services/ipblocksinfo/ipblocksinfo.go

```diff
@@ -137,6 +138,48 @@ func (s *IPBlocksInfoService) mergeIPCidrs(source []string, target []string) []s
 	return result
 }
 
+// mergeIPRanges merges target ranges into source CIDRs if not already covered by source.
+// Only considers IPv4, assumes no overlaps and all CIDRs are valid.
+// Assume there were no duplicate ranges in target,
+// None of the elements in target will be a subset of another element
```

Comment:
If needed, update the comments
```suggestion
// mergeIPRanges merges target ranges into source ranges if not already covered by source.
// Only considers IPv4, assumes no overlaps and all ranges are valid.
// Assume there were no duplicate ranges in target,
// None of the elements in target will be a subset of another element
```